strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java

```diff
@@ -362,4 +365,22 @@ void testRemoteRepositoryFormInvalidNegativeHttpConnectionPool()
 
         validateAndAssert(repositoryForm, 1, "A httpConnectionPool must be positive or zero.");
     }
+
+    private String getPatternLocalisedMessage(String expectedPattern)
+    {
+        String message;
+        try
+        {
+            final String translatedMessage = new PlatformResourceBundleLocator("org.hibernate.validator.ValidationMessages")
+                    .getResourceBundle(Locale.getDefault())
+                    .getString("javax.validation.constraints.Pattern.message");
```

Comment:
Please, format this code according to [our coding convention](https://strongbox.github.io/developer-guide/coding-convention.html). Thanks! :)

Reply:
Formatted

Reply:
Sorry, but that I don't think that this is well formated by using our code style formatter. It should be like this:
https://gist.github.com/ptirador/515a10e9bf752013e4bb8ff61121f9a1

Thanks.

Reply:
Commit bd9592cbfd364ba4480b464e73f73f660d0163af fixes this issue.

Reply:
Please take a look at what I said. It's not exactly well formated.